src/ui/appConfig.ts

```diff
@@ -1,4 +1,4 @@
-export const WAVESKEEPER_DEBUG = process.env.NODE_ENV !== 'production';
+export const KEEPERWALLET_DEBUG = process.env.NODE_ENV !== 'production';
```

Comment:
Does it make sense to keep the previous variable `WAVESKEEPER_DEBUG` as well?

Reply:
I think it doesn't matter